code-challenges.js

```diff
@@ -58,9 +127,25 @@ var notAString2 = 42
 
 var toonimals = [ { name: "Itchy", animal: "mouse" }, { name: "Stimpy", animal: "cat" }, { name: "Daffy", animal: "duck" }, { name: "Scratchy", animal: "cat" }, { name: "Ren", animal: "dog" }, { name: "Felix", animal: "cat" }]
 
+// declare a function that accepts an object as an argument
+const onlyCats = (object) => {
+  // return the result of filtering for only toonimals that are cats
+  return object.filter(value => value.animal === "cat")
```

Comment:
I'd be careful about the naming of our arguments being just 'objects' as technically, we're passing in an ARRAY of objects and not just the objects themselves. While the argument is arbitrary, as far as communication goes in our code, it might make more sense to name our argument 'array' or 'arrayOfObjects'. 

Again, nothing really wrong here, but just want to point out some better naming convention.